### DISCUSSION TITLE
Wavelog hosted?

### DISCUSSION THREAD

**Discussion by HB9EBE** [wavelog/wavelog#822] [Q&A] [answered]
Good morning, I am not a programmer and although I could manage to install Wavelog with my hosting service, I would greatly prefer not to have an additional piece of IT to maintain. Is there a hosted solution of Wavelog that takes care of installation and updates? Could not find any in the webpage or here in the discussion / WIKI.

**Comment by HB9HIL** [comment, marked as answer]
Dear @HB9EBE 

We do not offer any kind of commercial hosted service. We are just four guys having fun programming.

Vy 73 de HB9HIL

**Reply by HB9EBE** [reply]
Thanks for the lightening fast reply! I understand that you don't want the burden of hosting. Just thought someone else might offer a hosting service.... Good luck. Looks great!
Vy 73, Peter

**Comment by DK5EW** [comment]
Become a member of DARC they offer for members free Wavelog online

**Reply by HB9EBE** [reply]
Thank for letting me know. Checked the DARC site but could not find it.... (was a DARC member a while ago, in the early 1970s).

**Reply by DK5EW** [reply]
As I wrote you have to be a DARC  member to get access.

It`s coming up with the new DCL(next) as far I know think it`s still in testing phase but running very well for me.
https://www.darc.de/nachrichten/meldungen/aktuelles-details/news/dclnext-team-traf-sich-in-baunatal/

**Reply by HB9EBE** [reply]
Alles klar, wenn es das nächste DCL sein wird! Danke und 73

**Reply by DK5EW** [reply]
maybe missunterstanding Wavelog is allready running at DARC

**Reply by int2001** [reply]
Official launch will happen during September.
Yes, it is already running, but as a kind of get-well-Phase.

**Reply by HB9EBE** [reply]
Thank you!

**Comment by HB9EBE** [comment]
OK will try Tnx 73
